doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -492,18 +507,10 @@ Response:
 
 ## Entity by ID [/v2/entities/{entityId}{?attrs,options}]
 
-### Retrieve entity [GET /v2/entities/{entityId}{?attrs}]
++ Parameters
```

Comment:
However, if we move entityId definition under the ### sections, that makes FABRE and Apiary rendering to work?